### PR TITLE
API-16219/  doc server  create an endpoint for retrieving loast results list

### PR DESCRIPTION
**Pull request by mheusgit**
- Change implemented loast result list endpoint.
- Removed hardcoded sample result summary page
- Included support for pagination of result sets

### CODE REVIEW

File: src/loast/loastResult/loastResult.filter.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { LoastResultResponse } from '../interfaces/loast.output.interface';
+import LoastResultModel from './loastResult.model';
+
+const filterLoastResultResponse = (
```
Comment (dsteeve) [inline src/loast/loastResult/loastResult.filter.ts:4]: curious why this is called "filter".  What is being filtered with it?  
Is it more of a tally?

**Review by ediiotero — APPROVED** [review]
lgtm

**Review by dsteeve — APPROVED** [review]
I think this looks great Marc!
I think loastResult.tally vs loastResult.filter might be a better name.  thoughts?  
I generally think of filter as something that is filtering things out -- like "I only want errors".

File: src/loast/loastResult/loastResult.filter.ts (hunk 1 above)
Comment (mheusgit) [reply src/loast/loastResult/loastResult.filter.ts:4]: The naming comes from the other "filters" that message the data.  api.filter.ts, gatewayConfig.filter.ts, oas.filter.ts are some examples.  We can rename all of them if needed.

**Pull request merged**